apps/web/src/app/(private)/_data-access/index.ts

```diff
@@ -104,3 +105,23 @@ export const getDocumentByIdCached = cache(async (id: number) => {
 
   return document
 })
+
+export const getDocumentLogsWithMetadataCached = cache(
+  async ({
+    documentUuid,
+    commit,
+  }: {
+    documentUuid: string
+    commit: Commit
+  }) => {
+    const { workspace } = await getCurrentUser()
+    const docsScope = new DocumentLogsRepository(workspace.id)
+    const result = await docsScope.getDocumentLogsWithMetadata({
+      documentUuid,
+      draft: commit,
```

Comment:
Why is called `draft`?

Reply:
Because merged commits are always going to be included. This parameter will only affect the query when the provided commit is a draft.